packages/preview/diagraph/0.1.0/README.MD

```diff
@@ -0,0 +1,130 @@
+# diagraph
+
+A simple graphviz binding for typst using the new webassembly plugin system
+
+## Usage
+
+This plugin is quite simple to use, you just need to import it:
+
+```typ
+#import "@preview/diagraph:0.1.0": *
+```
+
+It allows you to render a graphviz dot string to a svg image:
+
+```typ
+#render("digraph { a -> b }")
+```
+
+For a simpler usage, you can setup a simple show rule:
+
+```typ
+#show: gaph-render
+```
+
+This allow you to draw a graphviz graph like this:
+
+```typ
+```dotrender
+digraph {
+    a -> b
+}
+`` `
+```
+
+You can see an example of this in the [example](https://github.com/Robotechnic/diagraph/tree/main/examples) folder.
+
+For more information about the graphviz dot language, you can check the [official documentation](https://graphviz.org/documentation/).
+
+## Functions definitions
+
+### render
+
+This function allow you to render a graphviz dot string to a svg image.
+
+```typ
+render(
+    dot: string,
+    engine: string,
+    width: auto relative,
+    height: auto relative,
+    fit: string
+)
+```
+
+| Parameter | Description | Default value |
+| :-------: | :---------: | :-----------: |
+| dot | The dot string to render | Required |
+| engine | The graphviz engine to use | dot |
+| width | The width of the svg | auto |
+| height | The height of the svg | auto |
+| fit | The fit mode of the svg | contain |
+
+All the optional parameters are the same as the one of images, you can check the [documentation](https://typst.app/docs/reference/visualize/image/) for more details.
+
+### raw-render
+
+This function is a show rule that allow you to use raw to render a graphviz dot string.
```

Comment:
it's not a show rule from what I can tell. is this a copy paste error?

Reply:
It's not a show rule, sorry. This is a copy-pase error.